dedupe/ckan_api.py

```diff
@@ -83,10 +83,10 @@ def request(self, method, path, **kwargs):
     def get(self, path, **kwargs):
         return self.request('GET', path, **kwargs)
 
-    def get_dataset(self, organization_name, identifier, is_collection, sort_order='asc'):
+    def get_dataset(self, organization_name, guid, is_collection, sort_order='asc'):
```

Comment:
Leave the identifier here as `identifier`. Add a new optional parameter similar to sort_order called `identifier_type` that defaults to `identifier`, and then use that parameter in place of the first `identifier` on line 88.

Reply:
Yes James, I already introduced the indentifier_type and trying to test it.